tools/kompos/src/ui-controller.ts

```diff
@@ -298,4 +298,31 @@ export class UiController extends Controller {
     this.view.onContinueExecution(act);
     this.vmConnection.sendDebuggerAction("return", act);
   }
+
+  public stepIntoMessage(actId: string) {
+    const activityId = getActivityIdFromView(actId);
```

Comment:
In all these methods, everything is duplicated but one string. Please don't do this!